lib/forest_liana/bootstraper.rb

```diff
@@ -35,7 +35,13 @@ def display_apimap
     private
 
     def generate_apimap
-      create_apimap
+      if Rails.env.production?
+        load_apimap_schema
+        create_apimap(false)
+      else
+        create_apimap
+      end
```

Comment:
```ruby
if Rails.env.development?
  create_apimap
else
  load_apimap_schema
  create_apimap(false)
end
```

Reply:
We are sure that `development` environment is the only environment where we want to have this behaviour.

Reply:
Even a more generic procedure:
```ruby
create_apimap if Rails.env.development? # we update the forestadmin.json file if in dev environment.
load_apimap_schema # we load the apimap from the file whatever the environment is.
```

Reply:
I think it's unefficient to load the schema while it has just been created. Why doing job we don't need to ?

Reply:
And the forestadmin-schema.json is not created at this moment (On development environment)